platform/android/export/export.cpp

```diff
@@ -1355,11 +1399,9 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/support_large"), true));
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/support_xlarge"), true));
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/opengl_debug"), false));
-
-		for (uint64_t i = 0; i < sizeof(launcher_icons) / sizeof(launcher_icons[0]); ++i) {
-			r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, launcher_icons[i].option_id, PROPERTY_HINT_FILE, "*.png"), ""));
-		}
-
+		r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "launcher_icon/xxxhdpi_192x192", PROPERTY_HINT_FILE, "*.png"), ""));
+		r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "launcher_adaptive_icon_foreground/xxxhdpi_432x432", PROPERTY_HINT_FILE, "*.png"), ""));
+		r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "launcher_adaptive_icon_background/xxxhdpi_432x432", PROPERTY_HINT_FILE, "*.png"), ""));
```

Comment:
Since the `launcher_*` strings are used in a couple locations, can you turn them into constants. This helps avoid subtle bugs/errors caused by typo when rewriting the string.

Reply:
I was divided between doing that or keeping the pattern of hardcoded strings for all the options. Constants it is then. :)